ethereum/statetransition/src/test/java/tech/pegasys/teku/statetransition/validation/BlockValidatorTest.java

```diff
@@ -210,4 +243,51 @@ void shouldReturnInvalidForBlockThatDoesNotDescendFromFinalizedCheckpoint() {
         blockValidator.validate(blockAndState.getBlock());
     assertThat(result).isCompletedWithValueMatching(InternalValidationResult::isReject);
   }
+
+  @TestTemplate
+  void shouldReturnAcceptOnCorrectExecutionPayloadTimestamp(SpecContext specContext) {
+    onlyMerge(specContext);
```

Comment:
nit: Would it make more sense to make this `specContext.onlyMerge()`?  Or probably more correct here would be `specContext.assumeMergeEnabled()` (or `assumeMergeActive`) so that it also runs on forks after the merge.